composeApp/src/androidInstrumentedTest/kotlin/org/ooni/probe/uitesting/screenshots/AutomateScreenshotsTest.kt

```diff
@@ -0,0 +1,130 @@
+package org.ooni.probe.uitesting.screenshots
+
+import android.Manifest
+import androidx.compose.ui.test.isDisplayed
+import androidx.compose.ui.test.junit4.createEmptyComposeRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.GrantPermissionRule
+import kotlinx.coroutines.test.runTest
+import ooniprobe.composeapp.generated.resources.Modal_EnableNotifications_Title
+import ooniprobe.composeapp.generated.resources.OONIRun_Run
+import ooniprobe.composeapp.generated.resources.Onboarding_AutomatedTesting_Title
+import ooniprobe.composeapp.generated.resources.Onboarding_Crash_Title
+import ooniprobe.composeapp.generated.resources.Onboarding_DefaultSettings_Button_Go
+import ooniprobe.composeapp.generated.resources.Onboarding_DefaultSettings_Title
+import ooniprobe.composeapp.generated.resources.Onboarding_PopQuiz_True
+import ooniprobe.composeapp.generated.resources.Onboarding_ThingsToKnow_Button
+import ooniprobe.composeapp.generated.resources.Onboarding_ThingsToKnow_Title
+import ooniprobe.composeapp.generated.resources.Onboarding_WhatIsOONIProbe_GotIt
+import ooniprobe.composeapp.generated.resources.Onboarding_WhatIsOONIProbe_Title
+import ooniprobe.composeapp.generated.resources.Res
+import ooniprobe.composeapp.generated.resources.app_name
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.ooni.probe.data.models.SettingsKey
+import org.ooni.probe.uitesting.helpers.clickOnTag
+import org.ooni.probe.uitesting.helpers.clickOnText
+import org.ooni.probe.uitesting.helpers.dependencies
+import org.ooni.probe.uitesting.helpers.onNodeWithContentDescription
+import org.ooni.probe.uitesting.helpers.onNodeWithText
+import org.ooni.probe.uitesting.helpers.preferences
+import org.ooni.probe.uitesting.helpers.start
+import org.ooni.probe.uitesting.helpers.wait
+import tools.fastlane.screengrab.Screengrab
+import tools.fastlane.screengrab.cleanstatusbar.BluetoothState
+import tools.fastlane.screengrab.cleanstatusbar.CleanStatusBar
+import tools.fastlane.screengrab.cleanstatusbar.MobileDataType
+import tools.fastlane.screengrab.locale.LocaleTestRule
+import kotlin.time.Duration.Companion.seconds
+
+@RunWith(AndroidJUnit4::class)
+class AutomateScreenshotsTest {
+    @Rule @JvmField
+    val localeTestRule = LocaleTestRule()
+
+    @get:Rule
+    val mRuntimePermissionRule: GrantPermissionRule =
+        GrantPermissionRule.grant(
+            Manifest.permission.CHANGE_CONFIGURATION,
+            Manifest.permission.WRITE_EXTERNAL_STORAGE,
+            Manifest.permission.READ_EXTERNAL_STORAGE,
+        )
```

Comment:
I thought it was doing this automatically, without the rule.

Reply:
It is possible to get rid of this.